ChangeLog.md

```diff
@@ -24,6 +24,8 @@ See docs/process.md for more on how version tagging works.
 - Using `-Oz` or `-Os` will no longer pass `-fno-inline-functions` to clang and
   instead rely on clang's normal inline heuristics for these optimization
   levels.  `-fno-inline-functions` can be passed explicitly if needed.
+- C++17 is now the default version of the C++ standard.  This is due to an
+  upstream change in llvm.  Use `-std=c++14` to revert to the previous default.
```

Comment:
technically I think the previous default was gnu++17, maybe this could say that or mention both.

Reply:
Do you mean gnu++14?

Reply:
Yes, I did mean gnu++14.